src/adagrams.js

```diff
@@ -1,15 +1,133 @@
-export const drawLetters = () => {
+
+//import { random } from "core-js/core/number";
+
+
+
+export const drawLetters = () =>{
   // Implement this method for wave 1
-};
+  let LETTER_POOL={
+    'A': 9,
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+  };
+  let remainingletterspool=[];
+  //create a varibale (let perhaps)parse out the letters
+  let letterChoice=Object.assign({},LETTER_POOL)
+  while (remainingletterspool.length < 10){
```

Comment:
Nice use of a while loop as opposed to a for loop here! In the context of your code, A while loop accounts for letters being drawn that aren't available.